node/Cargo.toml

```diff
@@ -33,12 +34,12 @@ sc-basic-authorship = '0.9.0'
 sc-cli = { features = ['wasmtime'], version = '0.9.0' }
 sc-client-api = '3.0.0'
 sc-consensus = '0.9.0'
+sc-keystore = '3.0.0'
```

Comment:
This is an unnecessary move
(keep minimal changes)

Reply:
agree.